run.py

```diff
@@ -935,24 +934,57 @@ def initialize_speaker(
         指定されたstyle_idの話者を初期化します。
         実行しなくても他のAPIは使用できますが、初回実行時に時間がかかることがあります。
         """
-        style_id = get_style_id_from_deprecated(style_id=style_id, speaker_id=speaker)
         engine = get_engine(core_version)
         engine.initialize_speaker_synthesis(style_id=style_id, skip_reinit=skip_reinit)
         return Response(status_code=204)
 
-    @app.get("/is_initialized_speaker", response_model=bool, tags=["その他"])
-    def is_initialized_speaker(
-        style_id: Optional[int] = Query(default=None),
-        speaker: Optional[int] = Query(default=None, deprecated=True),
+    @app.get("/is_initialized_style_id", response_model=bool, tags=["その他"])
+    def is_initialized_style_id(
+        style_id: int,
         core_version: Optional[str] = None,
     ):
         """
         指定されたstyle_idの話者が初期化されているかどうかを返します。
         """
-        style_id = get_style_id_from_deprecated(style_id=style_id, speaker_id=speaker)
         engine = get_engine(core_version)
         return engine.is_initialized_speaker_synthesis(style_id)
 
+    @app.post("/initialize_speaker", status_code=204, tags=["その他"], deprecated=True)
+    def initialize_speaker(
+        speaker: int,
+        skip_reinit: bool = Query(  # noqa: B008
+            False, description="既に初期化済みの話者の再初期化をスキップするかどうか"
+        ),
+        core_version: Optional[str] = None,
+    ):
+        """
+        こちらの利用は、非推奨です。`initialize_speaker` の利用を推奨しています。\n
```

Comment:
推奨するのは`initialize_style_id`ですかね
```suggestion
        こちらの利用は、非推奨です。`initialize_style_id` の利用を推奨しています。\n
```